Document/0x05e-Testing-Cryptography.md

```diff
@@ -216,6 +216,14 @@ Example initialization of DES algorithm, that is considered weak:
 ```Java
 Cipher cipher = Cipher.getInstance("DES");
 ```
+##### Block cipher encryption modes
+Do not use ECB encryption mode, it is basically raw cipher. For each block of input, you encrypt that block and get some output. The problem with this transform is that any resident properties of the plaintext might well show up in the ciphertext – possibly not as clearly – that's what blocks and key schedules are supposed to protect againt, but analyzing the patterns you may be able to deduce properties that you otherwise thought were hidden.
```

Comment:
Nice addition of the dangers of ECB mode. I think this could be more clear if you clearly separate sentences. For example,

> Do not use ECB encryption mode, it is basically raw cipher.

These are actually two sentences that should be separated by a period:

> Do not use ECB encryption mode. ECB mode is basically raw cipher.

Or they should be joined by indicating the relation:

> Since ECB encryption mode is basically raw cipher, it should not be used.